app/views/shared/_announcement.html.erb

```diff
@@ -1,10 +1,11 @@
 <%# to remove messages altogether, set this to empty string %>
-<% message = '' %>
+<% message = 'New BorrowDirect platform to launch 12/13/2022.' %>
 <% if Orangelight.read_only_mode %>
   <div class="col-12 alert alert-warning announcement">
     <div class="container">
       <p>
-      <%= message %> <%= Orangelight.read_only_message %>
+      <%= link_to message, "https://library.princeton.edu/news/general/2022-12-02/borrowdirect-new-platform-launch-december-13-2022" %> 
```

Comment:
The link isn't displaying for me on qa.  Shouldn't this `link_to` be in the `elsif` block below?

Reply:
That what explain why it didn't display. 😄